documentation/source/user_section/tutorials/gray-matter-segmentation/gm-wm-metric-computation/csa-gm-wm.rst

```diff
@@ -3,20 +3,39 @@ Using binary masks to compute CSA for gray and white matter
+
+In most cases, you will already have the cord segmentation, as often the GM and WM segmentation will be derived from the spinal cord segmentation. However, in case you do not have this segmentation, you can sum the grey and white matter masks, then optionally binarize the resulting cord mask if the GM/WM masks were non-binary.
+
+.. code::
+
+  sct_maths -i t2s_wmseg.nii.gz -add t2s_gmseg.nii.gz -o t2s_seg.nii.gz
+  sct_maths -i t2s_seg.nii.gz -bin 0.5
+
+Compute CSA
+-----------
+
+We can now pass both the GM/WM masks and the full cord segmentation to compute the slicewise CSA:
+
 .. code::
 
-   sct_process_segmentation -i t2s_wmseg.nii.gz -o csa_wm.csv -perslice 1 -angle-corr 0
+   sct_process_segmentation -i t2s_wmseg.nii.gz -o csa_wm.csv -perslice 1 -angle-corr-centerline t2s_seg.nii.gz
 
 .. code::
 
-   sct_process_segmentation -i t2s_gmseg.nii.gz -o csa_gm.csv -perslice 1 -angle-corr 0
+   sct_process_segmentation -i t2s_gmseg.nii.gz -o csa_gm.csv -perslice 1 -angle-corr-centerline t2s_seg.nii.gz
 
 :Input arguments:
    - ``-i`` : The input segmentation file.
    - ``-o`` : The output CSV file.
    - ``-perslice`` : Set this option to 1 to turn on per-slice computation.
-   - ``-angle-corr 0``: Normally, angle correction will be applied during ``sct_process_segmentation`` to account for scans where the spinal cord is positioned at an angle with respect to the superior-inferior axis. While this works well when the input is a full spinal cord segmentation, in this case we are instead providing GM/WM segmentations only. The shape of these segmentations can have a negative effect on so the estimation of the cord centerline, which in turn may cause the estimated angle to be incorrect. So, here we specify ``0`` to turn off angle correction.
-   - **Note:** Turning off angle correction is only safe to do if you know that your axial slices were acquired roughly orthogonal to the cord.
+   - ``-angle-corr-centerline``: Normally, angle correction will be applied during ``sct_process_segmentation`` to account for scans where the spinal cord is positioned at an angle with respect to the superior-inferior axis. While this works well when ``-i`` is a full spinal cord segmentation, in this case we are instead providing GM/WM segmentations to ``-i``. The irregular cross-sectional shape of these segmentations can have a negative effect on so the estimation of the cord centerline, which in turn may cause the estimated angle to be incorrect. So, here we explicitly provide a full spinal cord segmentation for angle correction purposes, to ensure accurate and consistent angle correction.
+   - **Note:** Alternatively, if you don't have a full-cord segmentation, you could turn off angle correction entirely using ``-angle-corr 0``, but this will only provide accurate results if you know that your axial slices were acquired roughly orthogonal to the cord.
```

Comment:
Maybe we should also specify that the curvature of the spinal cord affects this (if lrge FOV, it is impossible to be orthoginal through the entire FOV)